mungegithub/run.sh

```diff
@@ -1,24 +0,0 @@
-#!/bin/sh
-
-# Copyright 2015 The Kubernetes Authors All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-while true; do
```

Comment:
I'm not sure I understand. Before if the munger died this script would just launch it again (in 10 minutes).

When using the pod to launch directly if the munger died the container would be considered 'failed' and thus the kubelet would relaunch the container because the restart policy is OnFailure, Right?  So we get the same behaviour.

Except, I guess how you find the logs of the failure.